test/support/stub_helper.rb

```diff
@@ -11,26 +11,42 @@ def stub_env(env = {})
     end
   end
 
-  def stub_organization_membership_request(organization_id:, login:, successful:)
-    membership_url = %r{https://api.github.com/organizations/#{organization_id}/members/#{login}}
+  def stub_organization_membership_request(organization_id:, successful:)
+    orgs_url = %r{https://api.github.com/user/orgs}
 
     headers          = { 'Content-Type' => 'application/json', 'Cache-Control' => 'no-cach, no-store' }
-    default_response = { body: nil, headers: headers }
-
-    response = successful ? default_response.merge(status: 204) : default_response.merge(status: 404)
+    default_response = { headers: headers, status: 200 }
+
+    response = if successful 
```

Comment:
If the response wasn't successful, what do you think about returning an empty `Array` as the body?